src/Sdk/DTGenerated/Generated/TaskHttpClientBase.cs

```diff
@@ -324,20 +324,22 @@ public TaskHttpClientBase(Uri baseUrl, HttpMessageHandler pipeline, bool dispose
         /// <param name="scopeIdentifier">The project GUID to scope the request</param>
         /// <param name="hubName">The name of the server hub: "build" for the Build server or "rm" for the Release Management server</param>
         /// <param name="planId"></param>
+        /// <param name="jobId"></param>
         /// <param name="actionReferenceList"></param>
         /// <param name="userState"></param>
         /// <param name="cancellationToken">The cancellation token to cancel operation.</param>
         public virtual Task<ActionDownloadInfoCollection> ResolveActionDownloadInfoAsync(
             Guid scopeIdentifier,
             string hubName,
             Guid planId,
+            Guid jobId,
             ActionReferenceList actionReferenceList,
             object userState = null,
             CancellationToken cancellationToken = default)
         {
             HttpMethod httpMethod = new HttpMethod("POST");
             Guid locationId = new Guid("27d7f831-88c1-4719-8ca1-6a061dad90eb");
-            object routeValues = new { scopeIdentifier = scopeIdentifier, hubName = hubName, planId = planId };
+            object routeValues = new { scopeIdentifier = scopeIdentifier, hubName = hubName, planId = planId, jobId = jobId };
```

Comment:
will this new route without versioning break compat between the latest runner and old GHES.

Reply:
If it is a query parameter change without route value , would it still break the compat and do we need GHES testing in that case?

Reply:
I would suggest you always test GHES with any REST API changes.